playbooks/roles/custom_certificates/tasks/main.yml

```diff
@@ -87,3 +87,6 @@
       -out "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
   args:
     creates: "{{ custom_certificates_ca_directory_certs }}/{{ custom_certificates_server }}.crt"
+
+- include: issue.yml
+  when: custom_certificates_hostname is defined
```

Comment:
You should be able to use structured variables in boxes.yaml now! We switched from specifying host vars to extra vars (and for whatever reason, extra vars understands structured data).

https://github.com/theforeman/forklift/blob/master/Vagrantfile#L86
https://github.com/theforeman/forklift/blob/master/boxes.yaml.example#L33-L38